clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/description/GlobalResourcePropertyRegistry.java

```diff
@@ -52,6 +52,11 @@ public GlobalResourcePropertyRegistry(
         new KubernetesResourceProperties(defaultHandler, defaultHandler.versioned());
   }
 
+  public void setDefaultHandler(KubernetesUnregisteredCustomResourceHandler defaultHandler) {
```

Comment:
Would it make sense to check if the 'defaultHandler' is not null before creating the default properties?